src/Controllers.php

```diff
@@ -106,6 +113,40 @@ public function cheatsheet(Version $version)
         return $this->render('cheatsheet.twig', $twigVars);
     }
 
+    /**
+     * Controller for sitemap (single)
+     *
+     * @param Version $version
+     *
+     * @return mixed
+     */
+    public function sitemap(Version $version)
+    {
+        $twigVars = [
+            'menu' => $version->getMenu(),
```

Comment:
Could you use `$version->getPage('')` here instead. I'm actually trying to kill `getMenu`.

Maybe just change the key to `page` or `root` or `rootPage`.